test/common.mak

```diff
@@ -14,11 +14,13 @@ SRC:=src
 GENERATED:=./generated
 ROOT:=$(GENERATED)/$(OS)/$(BUILD)/$(MODEL)
 
+OPTIONAL_PIC:=$(if $(PIC),-fPIC,)
```

Comment:
I'd prefer to explicitly pass `OPTIONAL_PIC=$(OPTIONAL_PIC)` from the top makefile as is done for the other variables above.
See https://github.com/dlang/druntime/blob/4af3423c2898a1f61014dfbb03e8d8d6f5712cf2/posix.mak#L264-L266.
The fact that explict CLI arguments (like `PIC=1`) are passed to proper submake invocations is fairly subtle to rely on.